### DISCUSSION TITLE
Live demo Explorer not working - Internal Server Error #500

### DISCUSSION THREAD

**Discussion by V0rlov** [vesoft-inc/nebula#5856] [Q&A]
I want to assess Nebula features on the live demo environment: https://explorer.nebula-graph.io/schema/space/create
but the Create Space button gives Internal Server Error #500
Could you please advise if/when this can be resolved - many thanks

**Comment by QingZ11** [comment]
Sometimes the machine resources for this online service may become exhausted. The engineers at Nebula have likely dealt with this issue, and currently, the access is normal. You can try refreshing the page and clearing the cache before accessing it again.

If you have any further questions, feel free to update this thread.

**Reply by V0rlov** [reply]
Still same error - doesn't look like they've dealt with it - is there a better channel to raise this?

**Reply by QingZ11** [reply]
Sorry, I thought you were referring to an error when clicking the "create space" button, so when I was able to navigate to the next page normally after clicking, I assumed everything was working fine. I just tested it again, and after clicking, filling in the space information on the page, and then submitting by clicking the "create" button, it does not navigate to the next page. This issue has already been reported to the IT team for fixing. Thank you once again for your feedback, and I apologize for my previous misunderstanding.

**Reply by QingZ11** [reply]
After discussing with the product team, the "create space" permission is not public, and users are not allowed to create spaces. Therefore, the development team has made a fix, and this feature has been temporarily restricted.
